docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Building for TV
+description: A guide for building an Expo app for an Apple TV or Android TV target.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+> **Warning** TV development is available only in SDK 50 and later, and is still considered experimental. Not all Expo features and SDK modules are available on TV.
+
+React Native is supported on Apple TV and Android TV through the [React Native TV project](https://github.com/react-native-tvos/react-native-tvos).  It is not just for TV — it is a full fork of the core repo, and supports both phone and TV targets, including Hermes and Fabric support
+
+By taking an Expo project and using the React Native TV package as the `react-native` dependency, the project can be made to target either mobile (Android, iOS) or TV (Android TV, Apple TV) devices.
+
+The changes needed to the native Android and iOS files are minimal, and can be automated with a [config plugin](https://github.com/react-native-tvos/config-tv/tree/main/packages/config-tv) if you use [prebuild](/workflow/prebuild/). The following is a list of changes that the config plugin makes, which you can alternatively apply manually:
+
+- iOS:
+  - **ios/Podfile** is modified to target tvOS instead of iOS
+  - The Xcode project is modified to target tvOS instead of iOS
+  - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
+- Android:
+  - **AndroidManifest.xml** is modified
+  - The default phone portrait orientation is removed
+  - The required intent for TV apps is added
+  - **MainApplication.kt** is modified to remove unsupported Flipper invocations
```

Comment:
```suggestion
- Android:
  - **AndroidManifest.xml** is modified
  - The default phone portrait orientation is removed
  - The required intent for TV apps is added
  - **MainApplication.kt** is modified to remove unsupported Flipper invocations
- iOS:
  - **ios/Podfile** is modified to target tvOS instead of iOS
  - The Xcode project is modified to target tvOS instead of iOS
  - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
```

Let's apply the platform order formatting as per our writing style guide.